.github/workflows/test_osx_python.yml

```diff
@@ -45,13 +45,13 @@ jobs:
         source .ci/osx_ci.sh
         arm64_set_path_and_python_version ${{ matrix.python }}
         build_and_install_universal_kivy_sys_deps
-        install_kivy_test_run_pip_deps dev
```

Comment:
This doesn't do anything, does it?

Reply:
Absolutely nothing. Is probably something we left into during https://github.com/kivy/kivy/pull/7663 tests